### PR TITLE
Update UG, images and find command messages

### LINKED ISSUE #81: [PE-D][Tester A] Outdated screenshot in Quick Start section
Screenshot of the app included in the UG is not updated to the current version of the app

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: anthonyhoth/ped#1

### LINKED ISSUE #86: [PE-D][Tester B] Mismatch example and screenshots for find Command in UserGuide
Under finding eateries by name: find, the example given was `find pasta -r 3` however the screenshot and explanation below were for `find mala -r 2`, perhaps it would be good to try to coordinate the examples and screenshots shown so that it would cause less confusion for the users while reading the UserGuide.

-------------
Labels: `type.DocumentationBug` `severity.VeryLow`
original: bensohh/ped#2

### LINKED ISSUE #96: [PE-D][Tester B] Missing documents for Favourite and Unfavourite Commands in UserGuide
It was stated in the UserGuide that the Favourite/Unfavourite feature would be coming in later versions, however, this contradicts the help window upon typing in the `help` command, there are instructions listed for `fav/unfav` commands. Perhaps it would be good to standardize the documentation on the help window and the UserGuide.

-------------
Labels: `severity.Medium` `type.DocumentationBug`
original: bensohh/ped#5

### LINKED ISSUE #101: [PE-D][Tester A] UG does not explain wildcard -r for find command
It is unclear as to what `-r` does for find command unless the user looks at the given example screenshot and has to interpret that NUMBER is the random number of eateries to show. It would help to be clearer in the command descriptions.

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: anthonyhoth/ped#5

### LINKED ISSUE #109: [PE-D][Tester A] UG not updated to reflect Favourite/Unfavourite commands
App is able to `fav`/`unfav` eateries but UG does not provide such documentation

-------------
Labels: `severity.High` `type.DocumentationBug`
original: anthonyhoth/ped#8

### LINKED ISSUE #138: [PE-D][Tester C] find, findTag, findLocation, findCuisine Command Optional Parameters
Severity:

Medium

Reason:

In the UG, the -r parameter is optional, but in the command message, it's not. As a user, I wouldn't know which one is correct.

Same for findTag, findLocation, findCuisine

-------------
Labels: `type.FunctionalityBug` `severity.Medium`
original: cliftonfelix/ped#16

### PR DESCRIPTION
**Pull request by Francis-Tan**
Closes #81, #84, #87, #89,
#86,
#96, #109,
#101,
#138

### CODE REVIEW

File: docs/diagrams/ModelClassDiagram.puml
Comment (charleslimjh) [inline docs/diagrams/ModelClassDiagram.puml]: This should be changed to `Price` instead.

**Review by charleslimjh — CHANGES_REQUESTED** [review]
1. This is a good start but I think it's a bit premature to close issues 81, 84, 87, 89. 
The rest of the images have not been updated to reflect the new UI yet.

2. `ModelClassDiagram.png` still includes `Phone` which has been converted to `Price`.

4. This does not close issue #138, we would need to edit the help message in foodGuide to reflect that the -r delimiter is optional.

5. Can close #101 to an extent and definitely closes #96 and #109. Thanks!

We should minimally address points 1 and 2 before merging this PR.

File: docs/diagrams/ModelClassDiagram.puml
Comment (Francis-Tan) [reply docs/diagrams/ModelClassDiagram.puml]: Fair enough for 1, we should finalize the scroll bar first

**Review by charleslimjh — APPROVED** [review]
changes look good!

good point on fixing UI changes before updating UG. will wait till those changes are merged.

**Pull request merged**
